dash_auth/auth.py

```diff
@@ -90,11 +90,9 @@ def is_authorized_hook(self, func):
     def is_authorized(self):
         pass
 
-    @abstractmethod
```

Comment:
Deleting them would be a breaking change but happy to do that if @alexcjohnson is OK. Not sure what's the best way to mark them as deprecated otherwise.

Reply:
Unless I am mistaken, if users define / override these methods, it will do nothing at all. if anything, we should raise if these are defined in inherited classes

Reply:
That's a good point @olivier-lacroix - I don't actually see this being a breaking change, since deleting them is in effect _removing_ a restriction.

We could argue that #141 created a breaking change by no longer honoring the contents of these methods, in case someone made their own subclass that DID include them, but I view this as more of an implementation detail, not a feature people are likely to have used, and indeed I'm not aware of any complaints in the month that this has been out.

So I'd say let's go ahead and remove them throughout.